crud-module/index.js

```diff
@@ -0,0 +1,138 @@
+var Promise = require('bluebird'),
+    child_process = require('child_process'),
+    s = require('underscore.string'),
+    generators = require('yeoman-generator'),
+    mkdirp = require('mkdirp'),
+    log = require('../app/log');
+
+var exec = function (cmd) {
```

Comment:
Let's just extract this into a helper file and also refactor it out of the index.js